data/data/aws/variables-aws.tf

```diff
@@ -35,19 +32,16 @@ EOF
 
 variable "aws_master_root_volume_type" {
   type        = "string"
-  default     = "gp2"
   description = "The type of volume for the root block device of master nodes."
 }
 
 variable "aws_master_root_volume_size" {
   type        = "string"
-  default     = "120"
   description = "The size of the volume in gigabytes for the root block device of master nodes."
 }
 
 variable "aws_master_root_volume_iops" {
   type    = "string"
-  default = "400"
```

Comment:
There is also a default for this value in the master module, which can be removed.

https://github.com/openshift/installer/blob/7af6b141c34dd133f104b61b64c78d82e57f3d52/data/data/aws/master/variables.tf#L48-L52

Reply:
> There is also a default for this value in the master module, which can be removed.

Thanks, fixed with 580a34b -> 4dfa11f.